apps/studio/src/store/zustand/file.ts

```diff
@@ -0,0 +1,83 @@
+import { create } from "zustand";
+import { INITIAL_STATE } from "../getInitialState";
+
+export type File = {
+	hashDigest: string;
+	name: string;
+	content: string;
+	parent: string | null;
+};
+
+type FileState = {
```

Comment:
maybe FilesStore would be a better name? As this is not a state for a one single file